modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -660,4 +658,64 @@ function getIssueFields()
               ];
 
     return $result;
+}
+
+/**
+ * If issueID is passed retrieves issue data from database,
+ * othewise return empty issue data object
```

Comment:
I don't think it's a good idea to conflate 2 things like this, but it's probably no worse than it is.